README.md

```diff
@@ -22,9 +22,13 @@ Or install it yourself as:
 
 ## Usage
 
+### Fill watermark image method
+
+`fill_watermark_image(img, width, height, watermark_image_filename, gravity = "Center", horizontal_margin = 0, vertical_margin = 0, opacity = 100)`
```

Comment:
Thanks for your contribution, @dbaspi9 ! I feel like this section of the readme should perhaps be part of a new "API" heading instead of next to the `attachment_url` examples. What do you think?